src/Engine/StreamStorm/core/Playwright.py

```
@@ -178,6 +185,10 @@ async def find_and_click_element(
                     + selector
                 )
                 raise BrowserClosedError from e
+            
+        finally:
+            if check_brand_account:
+                return brand_account
```

Ruff (0.14.10)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)